src/core/components/table/AttendanceTable.vue

```diff
@@ -102,6 +132,9 @@ export default {
     noSlots () {
       return !this.course.slots.length;
     },
+    learners () {
+      return this.course.trainees.concat(this.unsubscribedLearners);
```

Comment:
Que penses-tu d'utiliser le spread `[...this.course.trainees, ...this.unsubscribedLearners]` ?

Reply:
done